.github/workflows/build-test.yml

```diff
@@ -103,3 +103,12 @@ jobs:
       - name: Build sources
         run: |
           cmake -Bbuild -Hiceoryx_meta -DBUILD_TEST=ON -DINTROSPECTION=OFF -DBINDING_C=ON -DEXAMPLES=ON && cmake --build build
+
+      # until the windows support is fully implemented and we can use the windows cmake targets
+      # we have to exclude the tests explicitly until everyone is running
+      - name: Run tests, excluding timing_tests
+        run: |
+          build\hoofs\test\Debug\hoofs_moduletests.exe --gtest_filter="-SignalHandler*:relativeptr*:Timer*:base_relative*:DeadlineTimer*:Thread*:Duration*:*TimingTest*"
```

Comment:
You can use here multiline strings with `\` as separator to make it more readable:
```bash
build\hoofs\test\Debug\hoofs_moduletests.exe \
--gtest_filter="-SignalHandler*:relativeptr*:Timer*:base_relative*:DeadlineTimer*:Thread*:Duration*:*TimingTest*"
```